lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -203,6 +203,16 @@ function paraneue_dosomething_preprocess_node_campaign(&$vars) {
   if (isset($vars['is_pitch_page'])) {
     $vars['show_persistent_signup'] = theme_get_setting('show_persistent_signup');
   }
+
+  drupal_add_js(
+    [
+      'dsUser' => [
+        'info' => dosomething_helpers_get_current_user(),
+        'activity' => NULL,
```

Comment:
Should we add a `@TODO` so we remember this is just a placeholder (and not something that's filled in by a hook elsewhere in the codebase)?